modules/microshift-cni-customization-matrix.adoc

```diff
@@ -39,10 +39,15 @@ The following table summarizes the status of networking features and capabilitie
 |IPsec encryption for intra-cluster communication|Not available|N/A
 
 |IPv6|Not available ^[5]^|N/A
+
+|Ingress router|Yes|Yes ^[6]^
+
+|Multiple networks plug-in|Yes|Yes
 |===
 
 1. If unset, the default value is set to the next immediate subnet after the service network. For example, when the service network is `10.43.0.0/16`, the `advertiseAddress` is set to `10.44.0.0/32`.
 2. You can use the multicast DNS protocol (mDNS) to allow name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
 3. There is no built-in transparent proxying of egress traffic in {microshift-short}. Egress must be manually configured.
 4. Setting up the firewalld service is supported by {op-system-ostree}.
-5. IPv6 is not available in any configuration.
+5. IPv6 is not supported. IPv6 can only be used by connecting to other networks with the {microshift-short} Multus CNI plugin.
+6. Using the {microshift-short} `config.yaml` configuration file.
```

Comment:
Typo? No verb. "6. Using the {microshift-short} `config.yaml` configuration file."